src/pages/policy/PolicyRightSidebar.jsx

```diff
@@ -41,29 +40,26 @@ function PolicyNamer(props) {
 }
 
 function SinglePolicyChange(props) {
-  const { startDateStr, endDateStr, parameterMetadata, value } = props;
+  const { startDateStr, endDateStr, parameterMetadata, value, paramLabel } = props;
+  const oldVal = getParameterAtInstant(parameterMetadata, startDateStr);
+  const oldValStr = formatVariableValue(parameterMetadata, oldVal);
   const newValueStr = formatVariableValue(parameterMetadata, value);
-  const startDate = moment(startDateStr).format("LL");
-  const endDate = moment(endDateStr).format("LL");
+
   return (
-    <div style={{ display: "flex", alignItems: "center" }}>
-      <div
-        style={{
-          width: 200,
-          textAlign: "right",
-          marginRight: 10,
-          flex: 3,
-          paddingRight: 5,
-          borderRightWidth: 2,
-          borderRightStyle: "solid",
-          borderRightColor: style.colors.DARK_GRAY,
-        }}
-      >
-        <div>from {startDate}</div>
-        <div>until {endDate}</div>
+    <div 
+      style={{ 
+        display: "flex", 
+        flexDirection: "column",
+        alignItems: "flex-start",
+        padding: "0 10%" }}
+    >
+      <div style={{ textAlign: "left" }}>
+          {paramLabel} from{' '}
```

Comment:
Could we add a `raise`/`lower`/`change` prefix here?

Reply:
@nikhilwoodruff Thank you. 
Are you asking me to name any styling prefix or for the value to be displayed? Could you please give an example.

Reply:
Ah, I meant adding a prefix to the text so that the message becomes:
`Raise [basic rate] from 20% to 40%`
or
`Lower [higher rate] from 40% to 35%`
or
`Change [refundable CTC] from false to true`

Depending on whether the new value is (a number and higher), (a number and lower), or (not a number/a boolean). Does that make sense?

Reply:
How about `[Disable/Enable] refundable CTC` instead of `Change refundable CTC from false to true`?